test/e2e/basic_test.go

```diff
@@ -101,7 +101,7 @@ func (data *TestData) testDeletePod(t *testing.T, podName string, nodeName strin
 		t.Fatalf("Expected 1 pod interface, got %d", len(podInterfaces))
 	}
 	ifName := podInterfaces[0].InterfaceName
-	podIP := podInterfaces[0].IP
+	podIPs := strings.Split(strings.ReplaceAll(podInterfaces[0].IP, " ", ""), ",")
```

Comment:
is this temporary? Do we intend to update the `Response` struct so that it supports an array of IPs?
From:
```go
type Response struct {
    // ...
    IP string
    // ...
}
```
To:
```go
type Response struct {
    // ...
    IPs []string
    // ...
}
```
Or (for backwards-compatibility):
```go
type Response struct {
    // ...
    IP string
    IPs []string
    // ...
}
```

Reply:
Good idea. Let's pick the first one.